app/stac_api/models.py

```diff
@@ -137,8 +139,18 @@ class Collection(models.Model):
     # furthermore GeoDjango and its functionality will be used for that.
     # TODO: overwrite items save() function accordingly
     # suggestions of fields to be auto-populated:
-    extent = models.JSONField(
-        default=get_default_extent_value, encoder=DjangoJSONEncoder, editable=False
+    # extent = models.JSONField(
+    #    default=get_default_extent_value, encoder=DjangoJSONEncoder, editable=False
+    # )
+    start_date = models.DateTimeField(editable=False, null=True, blank=True)
+    end_date = models.DateTimeField(editable=False, null=True, blank=True)
+    # the bbox field below has no meaning yet and was only introduced for testing
+    # while working on the temporal extent.
+    bbox = ArrayField(
+        ArrayField(models.FloatField(null=True, blank=True), null=True, blank=True),
+        null=True,
+        blank=True,
+        editable=False
```

Comment:
I think this is not necessary anymore since @rebert is using a geometry field, bbox can be deduced on serialization time from the geometry (or did you intend to 'cache' the calculation of the bbox from the geomtry in this field?)

Reply:
So far I did not really had anything in mind. The bbox field was just for test reasons (not filled with any functionality so far), as the "mother of all extents" has both, spatial and temporal. Just wanted to be able to have a correct extent in the end.

Reply:
Merging @rebert PR on this branch should solve it